ivadomed/config/config.json

```diff
@@ -40,12 +47,12 @@
         "test_fraction": 0.2
     },
     "training_parameters": {
-        "batch_size": 18,
+        "batch_size": 32,
```

Comment:
any reason for changing that? (unrelated to this PR)

Reply:
Again, no specific reason here as well. Waiting for 15 epochs is a lot lesser time than waiting for 100 epochs to know whether everything is running fine and not crashing anywhere during/after training. Well, 15 and not 5 because, segmentation outputs are not good for the latter (because too less training).